### PR TITLE
Apply results of finished /lend-by-category experiments

### PR DESCRIPTION
**Pull request by christian14b**
This pr removes all code related to fetching and tracking of the following experiments running on `lend-by-category` page:

ID: EXP-ML-Service-Bandit-LendByCategory
Winning version: control

ID: EXP-VUE-937-recommended-row-algo
Winning version: control/experiment off

ID: category_description
Winning version: variant-b

ID: add_to_basket_v2
Winning version: shown

ID: favorite_country
Winning version: control/experiment off

ID: hover_loan_cards
Winning version: variant-b

remaining only each experiment's winning version.

### CODE REVIEW

File: src/graphql/query/lendByCategory/lendByCategory.graphql [hunk 1]
```diff
@@ -4,30 +4,6 @@ query lendByCategory($basketId: String) {
...
-		}
-		addToBasketPopup: uiExperimentSetting(key: "add_to_basket_v2") {
-			key
-			value
-		}
-		favoriteCountryRow: uiExperimentSetting(key: "favorite_country") {
-			key
-			value
-		}
-		mlServiceBandit: uiExperimentSetting(key: "EXP-ML-Service-Bandit-LendByCategory") {
```
Comment (mcstover) [inline src/graphql/query/lendByCategory/lendByCategory.graphql:19]: I thought this was supposed to be evergreen? That we would also be passing the distribution and header so they could test variations on the algo?

**Review by mcstover — APPROVED** [review]
Great cleanup! Just want to verify the bandit item.

File: src/pages/Lend/LendByCategoryPage.vue [hunk 2]
```diff
@@ -282,24 +246,8 @@ export default {
 			return pageViewTrackData;
 		},
 		setRows(data) {
-			if (this.mlServiceBanditExpVersion !== null) {
```
Comment (mcstover) [inline src/pages/Lend/LendByCategoryPage.vue:285]: As @emuvente surfaced in our chats, this experiment is still active which ensure this query is always run resulting in our bandit algo determining the order of loan channels. This needs to remain in the code as we want to ensure the bandit algo still controls ordering.

File: src/pages/Lend/LendByCategoryPage.vue [hunk 3]
```diff
@@ -541,46 +463,20 @@ export default {
...
 			let expResults;
-
 			return client.query({
 				query: lendByCategoryQuery
 			}).then(({ data }) => {
 				// Get the array of channel objects from settings
 				rowData = readJSONSetting(data, 'general.rows.value') || [];
 				return Promise.all([
 					// experiment: GROW-330 Machine Learning Category row
-					client.query({ query: experimentQuery, variables: { id: 'EXP-ML-Service-Bandit-LendByCategory' } }),
```
Comment (mcstover) [inline src/pages/Lend/LendByCategoryPage.vue:573]: I think we need to keep this one

**Review by mcstover — COMMENTED** [review]
Getting very close. Just one comment to ensure the bandit is assigned.

File: src/pages/Lend/LendByCategoryPage.vue (hunk 3 above)
Comment (christian14b) [reply src/pages/Lend/LendByCategoryPage.vue:573]: You're right! I was keeping `EXP-VUE-937-recommended-row-algo` experiment. it's fixed now!

**Review by mcstover — APPROVED** [review]
Thank you @christian14b ! This is a great cleanup. Sorry for the thrash, we'll work on documenting this one exp better so it's not a confusing item.

**Review by emuvente — APPROVED** [review]
👏  🧹  Awesome!

**Review by eddieferrer — APPROVED** [review]

**Pull request merged**